roles/api-environment/files/express-api/run/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2014, 2019, Oracle and/or its affiliates. All rights reserved.
+#
+# ORACLE DOCKERFILES PROJECT
+
+FROM oraclelinux:7-slim
+ARG release=18
+ARG update=5
```

Comment:
Do we also wanna make them be variables in `default/main.yaml` and pass the value [here](https://github.com/osu-mist/ansible-roles/blob/1dd9b7637a55cbd3201f73fc42ad62cd8db1203f/roles/api-environment/tasks/express-api.yml#L52)?